src/WinGet.RestSource/Helpers/AppConfig/WinGetAppConfig.cs

```diff
@@ -0,0 +1,243 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="WinGetAppConfig.cs" company="Microsoft Corporation">
+//     Copyright (c) Microsoft Corporation. Licensed under the MIT License.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Microsoft.WinGet.RestSource.Helpers.AppConfig
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using global::Azure.Data.AppConfiguration;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.Configuration.AzureAppConfiguration;
+    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.FeatureManagement;
+    using Microsoft.Msix.Utils.Logger;
+    using Microsoft.WindowsPackageManager.Rest.Diagnostics;
+
+    /// <summary>
+    ///     WinGet App Configuration.
```

Comment:
>     [](http://example.com/codeflow?start=8&length=4)

nit: remove these spaces

Reply:
My editor keeps auto indenting all of them. :cry: 
I'll fix them.

Reply:
I see it in other places in this file. Is going to be our standard in this project?